src/processes/helper/userIdsToProcess.js

```diff
@@ -23,87 +42,88 @@ const findF2UserIds = async (userId) => {
   const f2UserIds = await UserFollowUser.findAll({
     where: {
       [Op.and]: [
-        { user_id_follower: { [Op.in]:followedUserIds} },
-        { user_id_followed: { [Op.notIn]: excludeUserIds } }
+        {user_id_follower: {[Op.in]: followedUserIds}},
+        {user_id_followed: {[Op.notIn]: excludeUserIds}}
       ]
     },
     limit: 1000,
     attributes: ['user_id_followed'],
-    raw : true
-  }).then( users => users.map(user => user.user_id_followed));
+    raw: true
+  }).then((users) => users.map((user) => user.user_id_followed));
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code is using a limit of 1000 for the number of users to fetch. This could potentially miss some users if there are more than 1000 users that match the criteria. If the application can handle larger data sets, consider removing this limit or increasing it.



<!-- This is an auto-generated comment by OSS CodeRabbit -->